docs/docs/building-a-contact-form.md

```diff
@@ -0,0 +1,140 @@
+
+In order to begin leveraging Formspree's features, you must add a form action directing the http POST method to the Formspree API (substituting your chosen email), as well as changing the `name` attribute of the email input to `name="_replyto"`.
+
+```jsx:title=src/pages/contact.js
+<form method="post" action="https://formspree.io/email@domain.tld">
+  ...
+  <label>
+    Email
+    <input type="email" name="_replyto" />
+  </label>
+  ...
+</form>
+```
+
+Once you've made the changes you can submit your own form for the first time and register using the email Formspree will send you, and all subsequent form submissions will be sent to your email address. You can find more information on the registration process or setup [on their website](https://formspree.io/).
+
+### Run Your Own Server
+
+If your form data requires a significant amount of business logic to handle, creating your own service might make the most sense. The most popular solution to this is writing an HTTP server - this can be done in many languages including PHP, Ruby, GoLang, or in our case Node.js with [Express](https://expressjs.com/).
+
+An initial implementation of a server using express, body-parser, and nodemailer may look like this:
+
+```javascript:title=handleForm.js
+const bodyParser = require("body-parser")
+const express = require("express")
+const nodemailer = require("nodemailer")
+
+const app = express()
+app.use(bodyParser.urlencoded())
+
+const contactAddress = "hey@yourwebsite.com"
+
+const mailer = nodemailer.createTransport({
+  service: "Gmail",
+  auth: {
+    user: process.env.production.GMAIL_ADDRESS,
+    pass: process.env.production.GMAIL_PASSWORD,
+  },
+})
+
+app.post("/contact", function(req, res) {
+  mailer.sendMail(
+    {
+      from: req.body.from,
+      to: [contactAddress],
+      subject: req.body.subject || "[No subject]",
+      html: req.body.message || "[No message]",
+    },
+    function(err, info) {
+      if (err) return res.status(500).send(err)
+      res.json({ success: true })
+    }
+  )
+})
+
+app.listen(3000)
+```
+
+This initial implementation listens for POST requests to `/contact`, and sends you an email with the submitted form data. You can deploy this server with services such as [Now](https://zeit.co/now).
+
+Once deployed, note the url of the deployment (something like `my-project-abcd123.now.sh`), and use it as your form action:
+
+```jsx:title=src/pages/contact.js
+<form method="post" action="my-project-abcd123.now.sh/contact">
+  ...
+</form>
+```
+
+Now, all subsequent form submissions will be sent to your email address!
+
+For an in-depth guide on running your own mail server, you can refer to [this awesome guide by DataFire](https://medium.com/datafire-io/simple-backends-four-ways-to-implement-a-contact-us-form-on-a-static-website-10fc430984a4).
```

Comment:
Are there any security/spam implications we should mention here?

Reply:
I found an article that goes into some of the detail of complexity and spam implications, though it isn't messaged from the perspective of a static site: https://www.digitalocean.com/community/tutorials/why-you-may-not-want-to-run-your-own-mail-server